### PR TITLE
Add locales

### PR DESCRIPTION
**Pull request by p1d1d1**
it, it-CH, fr-CH, de-CH

### CODE REVIEW

File: src/locales/it/texts.json [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+  },
+  "leaflet": {
+    "close": "Chiudi",
+    "noFeatureProperties": "Non ci sono altri dati disponibili per questo Elemento.",
+    "stayLayer": {
+      "error": "Non è stato possibile aggiungere i dati alla mappa"
+    },
+    "zoom": {
+      "in": {
+        "description": "Zoom in",
```
Comment (m-mohr) [inline src/locales/it/texts.json:128]: "Zoom in / out" are the same in Italian?

File: src/locales/it/texts.json [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+    "andMore": "e {count} in più"
+  },
+  "open": "Apri",
+  "pagination": {
+    "first": "« Primo",
+    "last": "Ultimo »",
+    "next": "Prossimo ›",
+    "previous": "‹ Precedente"
+  },
+  "poweredBy": "Powered by {link}",
```
Comment (m-mohr) [inline src/locales/it/texts.json:157]: This is missing a translation, right?

File: src/locales/it/texts.json [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+  "protocol": {
+    "azure": "Microsoft Azure",
+    "file": "file system (local)",
```
Comment (m-mohr) [inline src/locales/it/texts.json:160]: Is this missing a translation?

File: src/locales/it/texts.json (hunk 1 above)
Comment (p1d1d1) [reply src/locales/it/texts.json:128]: Could be. I'll change it

File: src/locales/it/texts.json (hunk 2 above)
Comment (p1d1d1) [reply src/locales/it/texts.json:157]: I'll change it

File: src/locales/it/texts.json (hunk 3 above)
Comment (p1d1d1) [reply src/locales/it/texts.json:160]: yep

**Pull request merged**